data_processing.ipynb

```diff
@@ -163,13 +168,14 @@
    "source": [
     "# Define useful variables\n",
     "\n",
-    "path_data = os.getcwd()\n",
+    "path_data = os.getcwd() #works if and only if this is run first and once. Otherwise it will pick up whatever the current directory is if someone wanted to rerun it\n",
```

Comment:
That's the motivation for #33

No need to add a comment in the notebook (better to fix #33 instead)